packages/transport/test/modules/account/IdentityDeletionProcessController.test.ts

```diff
@@ -0,0 +1,116 @@
+    });
+
+    test("should start an Identity deletion process from the Backbone admin API", async function () {
+        const activeIdentityDeletionProcess = await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+        expect(activeIdentityDeletionProcess.status).toBe(IdentityDeletionProcessStatus.WaitingForApproval);
+    });
+
+    test("should not start a second Identity deletion process from the Backbone admin API", async function () {
+        await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+        await expect(TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account)).rejects.toThrow("Request failed with status code 400");
+    });
+
+    test("should not start an Identity deletion process from the Backbone admin API if one was already started by the user", async function () {
+        await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        await expect(TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account)).rejects.toThrow("Request failed with status code 400");
+    });
+
+    test("should get the waiting Identity deletion process", async function () {
+        const cancelledIdentityDeletionProcess = await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        await account.identityDeletionProcess.cancelIdentityDeletionProcess(cancelledIdentityDeletionProcess.id.toString());
+        const waitingIdentityDeletionProcess = await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+        const result = await account.identityDeletionProcess.getWaitingIdentityDeletionProcess();
+        expect(waitingIdentityDeletionProcess.toBase64()).toBe(result!.toBase64());
+        expect(waitingIdentityDeletionProcess.status).toBe(IdentityDeletionProcessStatus.WaitingForApproval);
+    });
+
+    test("should approve an Identity deletion process that was started from the Backbone admin API", async function () {
+        const startedIdentityDeletionProcess = await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+
+        const result = await account.identityDeletionProcess.approveIdentityDeletionProcess(startedIdentityDeletionProcess.id.toString());
+        expect(result).toBeDefined();
+        expect(result.id.toString()).toBe(startedIdentityDeletionProcess.id.toString());
+        expect(result.status).toBe(IdentityDeletionProcessStatus.Approved);
+
+        const approvedIdentityDeletionProcess = await account.identityDeletionProcess.getApprovedIdentityDeletionProcess();
+        expect(approvedIdentityDeletionProcess).toBeDefined();
+        expect(approvedIdentityDeletionProcess!.status).toBe(IdentityDeletionProcessStatus.Approved);
+        expect(approvedIdentityDeletionProcess!.id.toString()).toBe(result.id.toString());
+    });
+
+    test("should reject an Identity deletion proces that was started from the Backbone admin APIs", async function () {
```

Comment:
"Identity deletion process" instead "Identity deletion proces", "Backbone admin API" instead of "Backbone admin APIs"